cmd/snap-confine/udev-support.c

```diff
@@ -116,6 +116,34 @@ static void sc_udev_allow_nvidia(sc_device_cgroup *cgroup)
 	}
 }
 
+/** Allow access to hybris devices.
+ *
+ * Required by Halium-based GNU/Linux adaptations to make use of certain device nodes.
+ *
+ * Note: Binder devices on newer Android kernels reside inside of their own binderfs mountpount.
+ **/
+static void sc_udev_allow_hybris(sc_device_cgroup *cgroup)
+{
+	struct stat sbuf;
+
```

Comment:
nitpick: would this be easier to read if it was written as something like the following (note untested):

```c
const char *paths[] = {"/dev/binderfs/binder", "/dev/binderfs/hwbinder", "/dev/binder", "/dev/hwbinder"};
for (int i = 0; i < sizeof(paths)/sizeof(paths[0]); i++) {
    struct stat sbuf;
    if (stat(paths[i], &sbuf) == 0) {
        sc_device_cgroup_allow(cgroup, S_IFCHR, major(sbuf.st_rdev),
				       minor(sbuf.st_rdev));
    }
}
```

Reply:
Thanks, based it on your template. Handled in d5a85d6